pkg/params/info/events.go

```diff
@@ -20,6 +21,9 @@ type Event struct {
 	// a push or a pull_request
 	TriggerTarget string
 
+	// Target PipelineRun, the target PipelineRun user request. Used in incoming webhook
+	TargetPipelineRun string
```

Comment:
do we want to use https://github.com/openshift-pipelines/pipelines-as-code/blob/99afbaa5342f8559227580fa97bef44a8522b1f8/pkg/params/info/events.go#L59 for this?
or need a new one